packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -441,6 +442,24 @@ function _processSpokeResponse(botKey, spokeResponse, validOutputs, errorOutputs
           spokeResponse.reason.response.data.execResponse),
       botIdentifier: botKey,
     };
+  } else if (spokeResponse.value && spokeResponse.value.execResponse && spokeResponse.value.execResponse.stdout == "") {
```

Comment:
these are the additional failure mode checks we look for: `stdout` is blank or the log does not contain "started"